ampoule/pool.py

```diff
@@ -377,15 +383,15 @@ def _cb(_):
         return defer.DeferredList(l).addCallback(_cb)
 
     def dumpStats(self):
-        log.msg("ProcessPool stats:")
-        log.msg('\tworkers: %s' % len(self.processes))
-        log.msg('\ttimeout: %s' % (self.timeout))
-        log.msg('\tparent: %r' % (self.ampParent,))
-        log.msg('\tchild: %r' % (self.ampChild,))
-        log.msg('\tmax idle: %r' % (self.maxIdle,))
-        log.msg('\trecycle after: %r' % (self.recycleAfter,))
-        log.msg('\tProcessStarter:')
-        log.msg('\t\t%r' % (self.starter,))
+        log.info(u'ProcessPool stats:')
```

Comment:
Would you mind putting this together into one big log message, rather than spreading it out across a whole bunch of different ones?  Conceptually, what's happening here is a dump of stats at a particular moment in time; they should all be associated with each other.  The repeated logging calls were just there trying to get the message to format in a particular way, which shouldn't be the log emitter's concern.